src/griddeddata.cpp

```diff
@@ -22,10 +22,12 @@ GridAxis::GridAxis(std::vector<double> grid_vector, Method extrapolation_method,
       interpolation_method(interpolation_method),
       extrapolation_limits(std::move(extrapolation_limits)) {
   if (grid.size() == 0) {
-      showMessage(MsgLevel::MSG_ERR, "Cannot create a GridAxis from a zero-length vector.");
+    showMessage(MsgLevel::MSG_ERR, "Cannot create a GridAxis from a zero-length vector.");
   }
   check_grid_sorted();
   check_extrap_limits();
+  // Probably need to add something like check_extrapolation_methods() here; is
```

Comment:
I'm not a huge fan of having comments in the code that are essentially asking questions. If it is not descriptive, I feel it should be removed. This may be something we actually need to put in our git policies. Can these be removed?